src/index.js

```diff
@@ -169,6 +172,16 @@ export default class AuthnWidget {
     }
   }
 
+  postDeviceProfileAction() {
+    let idw = document.getElementById('idw');
+    if (idw) {
+      setTimeout(() => {
+        this.store.dispatch('POST_FLOW', 'continueAuthentication', '{}');
+      },
+      3000);
```

Comment:
Curious where the 3 seconds came from. Are we just giving the javascript just enough time to run? or we want the user to see a spinner for a little bit?

Reply:
The adapter actually has a default 5 seconds profiling time if not using the widget (giving the javascript enough time to run). It's 3 seconds here is because the profiling time seems a lot longer with the widget for some reason (possibly due to some loading inside the widget itself?), but I can change it back to 5 seconds for consistency.

Reply:
ah I see. If its a setting in the adapter that admins can set, do you think we need to expose it in the state payload so the client would know the configured value?

Reply:
Agree, I'll add that in the state payload. 

Would you happen to know if there is a more straight forward way I can use the device profiling time value from the payload in index.js or will I have to add that as an attribute to one of the div in the handlebar and read that in index.js?

Reply:
You would probably have to store it in the state somehow.

It might actually be easier to move the logic to the handlebar template. There you could put a form with a hidden "continueAuthentication" button that gets auto-submitted. You'll have access to that variable in the handlebar.

Reply:
I wasn't able to find a good way to have logic handled just in the handlebar template, so I just stored the state as data and read the value in index.js.